FilterPlugin/src/main/java/io/antmedia/filter/FilterAdaptor.java

```diff
@@ -459,7 +454,7 @@ public synchronized Result createOrUpdateFilter(FilterConfiguration filterConfig
 					logger.info("Output stream:{} is same as input stream so no new customBroadcast will be created for filterId:{}", streamId, filterId);
 				}
 				else {
-					IFrameListener broadcast = app.createCustomBroadcast(streamId);
+					IFrameListener broadcast = app.createCustomBroadcast(streamId, filterConfiguration.getVideoOutputHeight(), filterConfiguration.getVideoOutputBitrate());
```

Comment:
I think it's not backward compatible. If an user does not send video width/height in the REST request. It will fail. 
 If yes, please make it compatbile. If no, please correct me.